This can be removed on the next integrate as it already exists in upstream.
diff --git a/lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp b/lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp
--- a/lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp
+++ b/lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp
@@ -21,7 +21,7 @@ namespace {
 static int getMMAVersionSafe(int computeCapability, DotOp op) {
   // List supported mma version in order of preference.
   SmallVector<int> versionsSupported;
-  if (computeCapability < 80) {
+  if (computeCapability < 75) {
     versionsSupported = {1};
   } else if (computeCapability < 90) {
     versionsSupported = {2};
